src/test/java/eu/dissco/backend/TestUtils.java

```diff
@@ -0,0 +1,54 @@
+package eu.dissco.backend;
+
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import eu.dissco.backend.domain.JsonApiData;
+import eu.dissco.backend.domain.JsonApiLinks;
+import eu.dissco.backend.domain.JsonApiWrapper;
+import eu.dissco.backend.domain.User;
+
+public class TestUtils {
+
+  public static final String USER_ID_TOKEN = "e2befba6-9324-4bb4-9f41-d7dfae4a44b0";
+  public final static String TYPE = "users";
+  public final static String FORBIDDEN_MESSAGE =
+      "User: " + USER_ID_TOKEN + " is not allowed to perform this action";
+
+  public static final ObjectMapper MAPPER = new ObjectMapper().findAndRegisterModules();
+
+  public static JsonApiWrapper givenUserResponse() {
+    return new JsonApiWrapper(givenJsonApiData(),
+        new JsonApiLinks("https://sandbox.dissco.tech/api/v1/users/" + USER_ID_TOKEN));
+  }
+
+  public static JsonApiWrapper givenUserRequest() {
+    return givenUserRequest(USER_ID_TOKEN);
+  }
+
+  public static JsonApiWrapper givenUserRequestInvalidType() {
+    return new JsonApiWrapper(
+        new JsonApiData(USER_ID_TOKEN, "annotations", MAPPER.valueToTree(givenUser())), null);
+  }
+
+  public static JsonApiWrapper givenUserRequest(String id) {
+    return new JsonApiWrapper(givenJsonApiData(id), null);
+  }
+
+  public static JsonApiData givenJsonApiData() {
+    return givenJsonApiData(USER_ID_TOKEN);
+  }
+
+  public static JsonApiData givenJsonApiData(String id) {
+    return new JsonApiData(id, TYPE, MAPPER.valueToTree(givenUser()));
+  }
+
+  public static User givenUser() {
+    return new User(
+        "Test",
+        "User",
+        "test@gmail.com",
+        "https://orcid.org/0000-0002-XXXX-XXXX",
+        "https://ror.org/0566bfb96"
```

Comment:
Does the ROR foundation maybe have a default test ROR identifier? Instead of using the Naturalis one. Probably not very important however...

Reply:
No this identifier isn't called anywhere so don't see an issue there. Could also use "XXX" but this way it at least looked like an ROR